packages/grid/x-data-grid/src/components/menu/columnMenu/GridColumnMenuProps.ts

```diff
@@ -1,10 +1,16 @@
 import * as React from 'react';
 import { GridColDef } from '../../../models/colDef/gridColDef';
+import { GridColumnMenuRootProps } from '../../../hooks/features/columnMenu';
 
-export interface GridColumnMenuProps extends React.HTMLAttributes<HTMLUListElement> {
+export interface GridColumnMenuContainerProps extends React.HTMLAttributes<HTMLUListElement> {
   hideMenu: (event: React.SyntheticEvent) => void;
   currentColumn: GridColDef;
```

Comment:
Maybe use v6 as an opportunity to improve the API consistency?
We can rename `currentColumn` here to `colDef` instead to make it consistent with `GridCellParams` and other interfaces where we pass the column definition.
Same with the `GridColumnHeaderItemProps['column']` - we can rename `column` to `colDef`, so the prop is the same for both column menu and column menu item components.
What do you think?

Reply:
I think that's a good idea, previously I was thinking to make `column` and `currentColumn` consistent, but `colDef` seems better

Reply:
By the way should we also change DataGrid prop `columns` to `colDefs` to make it consistent everywhere?

Reply:
I think the `columns` prop is fine.
My main concerns here were callbacks and components that we pass the column definition to.

Reply:
Alright, we're good then 👍